Mage/src/main/java/mage/game/permanent/token/TokenImpl.java

```diff
@@ -258,6 +263,17 @@ private static void putOntoBattlefieldHelper(CreateTokenEvent event, Game game,
             Token token = entry.getKey();
             int amount = entry.getValue();
 
+            // check if token needs to be attached to something
+            Permanent permanentAttachedTo;
+            if (attachedTo != null) {
+                permanentAttachedTo = game.getPermanent(attachedTo);
+                if (permanentAttachedTo == null || permanentAttachedTo.cantBeAttachedBy(token, source, game, true)) {
+                    continue;
```

Comment:
That's code will ignore a token creation on wrong `attachedTo` param (will continue to next token).

If it's a wrong code usage situation then must raise exception.
If it's a game's use case by a rules then token must be created or users must be informed about miss token due non valid attach target.

Reply:
it's supposed towel that way, you're right that users should probably be notified though

Reply:
Found related rules for it, can be ref in code:

> 303.4g If an Aura is entering the battlefield and there is no legal object or player for it to enchant, the Aura remains in its current zone, unless that zone is the stack. In that case, the Aura is put into its owner’s graveyard instead of entering the battlefield. **If the Aura is a token, it isn’t created**.